### DISCUSSION TITLE
How use Want_Angular_Dist and Want_SAMRML_RM option in RECONR

### DISCUSSION THREAD

**Discussion by kernie** [njoy/NJOY2016#318] [Q&A] [answered]
Hello,

I wanted to play around with those options in reconr.f90:148 , but setting those to true leads to a segfault with ENDF/B VIII U-238 file.

Do I have to change something in the ENDF file aswell?

Thank you for any hint or literature recommendation...!

**Comment by jchsublet** [comment, marked as answer]
Try Pu239 VIII.1b1, not U238 as you need an LRF7 evaluation. You may want to have a peak at Nancy's 2003 CSWEG RML presentation, also https://www.oecd-nea.org/dbdata/nds_jefdoc/jefdoc-2223.pdf, then decide if..

[jefdoc-2223.pdf](https://github.com/njoy/NJOY2016/files/13164319/jefdoc-2223.pdf)

**Comment by kernie** [comment]
Thank you @jchsublet for providing this "entry point" into the matter!
